### DISCUSSION TITLE
Build a pdf

### DISCUSSION THREAD

**Discussion by Zerotask** [retypeapp/retype#270] [Ideas]
Building a website is great but if you want to share your documentation with customers it's always great to send a simple pdf file instead of a zip with a whole website.

Therefore I suggest an option for the build command to create a pdf instead of a website.

**Comment by fabriciomurta** [comment]
Hi! Please refer to #84, #85 and #104. They are all related to what you're asking. I am going forward and labeling this as duplicate as it is basically the same of #84. Let us know if you don't agree.
